gap/WordToString.g

```diff
@@ -0,0 +1,8 @@
+WordToString := function(alphabet, word)
```

Comment:
You should remove this file completely, `git rm gap/WordToString.g`

Reply:
Actually you re-introduced it when you did a merge.